python/ray/experimental/iter.py

```diff
@@ -1,6 +1,7 @@
 from typing import TypeVar, Generic, Iterable, List, Callable, Any
 
 import ray
+import random
```

Comment:
For imports, please follow the order described in http://google.github.io/styleguide/pyguide.html#313-imports-formatting

*Imports are always put at the top of the file, just after any module comments and docstrings and before module globals and constants. Imports should be grouped from most generic to least generic:*

*Python future import statements. For example:*

    from __future__ import absolute_import
    from __future__ import division
    from __future__ import print_function

*Python standard library imports. For example:*

    import sys

*third-party module or package imports. For example:*

    import tensorflow as tf

*Code repository sub-package imports. For example:*

    from otherproject.ai import mind